docker/build/mssql-init/sql/ResetDemoUsers.sql

```diff
@@ -0,0 +1,32 @@
+USE [Sitecore.Core]
+
+DECLARE @UserId uniqueidentifier, @UserName nvarchar(256)
+
+DECLARE uc CURSOR FOR
+SELECT UserId, UserName
+FROM [aspnet_Users]
+WHERE username in ('sitecore\minnie', 'sitecore\pete', 'sitecore\demoadmin', 'sitecore\Content Author', 'sitecore\Marketer')
```

Comment:
There are some users we do not have in this demo yet that are listed here. Should they be removed for now?